src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -237,20 +190,20 @@ public ConnectNewHeadersResult HeadersPresented(INetworkPeer peer, List<BlockHea
             }
 
             if (triggerDownload && (connectNewHeadersResult.DownloadTo != null))
-                this.DownloadBlocks(connectNewHeadersResult.ToArray(), this.ProcessDownloadedBlock);
+                this.DownloadBlocks(connectNewHeadersResult.ToArray(), this.OnBlockDownloaded);
```

Comment:
`OnBlockDownloaded`. Usually smth that strats with `On` is an event. In our case it's a normal method so I think old name was less missleading

Reply:
Agreed, ProcessDownloadedBlock was a callback not an event.

Reply:
This has been reverted.